masu/api/download.py

```diff
@@ -34,13 +34,17 @@ class DownloadView(View):
     def dispatch_request(self):
         """Packages response for class-based view."""
         orchestrator = Orchestrator()
-        files = orchestrator.prepare_curs()
-        orchestrator.process_curs()
+        orchestrator.prepare_curs()
+        # orchestrator.process_curs()
 
-        downloaded_results_msg = 'Files downloaded and processed: {}'.format(str(files))
-        logger.info(downloaded_results_msg)
+        # downloaded_results_msg = 'Files downloaded and processed: {}'.format(str(files))
+        # logger.info(downloaded_results_msg)
+
+        # response = {
+        #     'files': files,
+        # }
 
         response = {
```

Comment:
This is where I was saying we could block until async results are complete.  Alternatively, we could also use flower to verify completion or make other test endpoints to get the status of pending results.  If we went the test endpoint route we would return some unique identifier that would be the path for status.

```
/api/v1/download => 200 response body: { [pending_async_req1, pending_async_req2,...]}
/api/v1/download/status/{} => i.e. /api/v1/download/status/pending_async_req1 => 200 response body: { 'files': files} or whatever
```

Reply:
Celery provides a results backend where it reports the results of each task. 

If we need to check the results of the task, we should use the facility provided by Celery.